optuna/samplers/_cmaes.py

```diff
@@ -60,6 +60,7 @@ def objective(trial):
             study.optimize(objective, n_trials=20)
 
     Please note that this sampler does not support CategoricalDistribution.
+    (However, `suggest_discrete_uniform` and `suggest_int` are supported.)
```

Comment:
I think it's more consistent to use the name of distributions.
```suggestion
    (However, DiscreteUniformDistribution and Int(Log)Distribution are supported.)
```

Reply:
@HideakiImamura 
Can we mention both `suggest` and `distribution` since a user samples a hyperparameter via either depending on interface: define-and-run or define-and-run?

Reply:
I will wait for an answer from @HideakiImamura

As a user, I am more familiar with the suggest_ interface.

Reply:
We could add both, if you like.


However, DiscreteUniformDistribution (`suggest_discrete_uniform`) and Int(Log)Distribution (`suggest_int`) ...

Reply:
Please check the revision

Reply:
Thanks for the update. The current version looks good to me.

Reply:
Thank you for your swift action!